### PR TITLE
[hold][Feature Fix] Fix escaped html appearing in user job and school information.

### PR DESCRIPTION
**Pull request by brandonPurvis**
# Purpose
Escaped html characters appears in a users job and school information both in search results and on the profile page.

screen shot 2015-07-09 at 11 00 16 am

screen shot 2015-07-09 at 11 06 15 am

# Changes
* Unescape html in user document before storing in elastic search.
* Unescape html in the results of serialize_job and serialize_school.

### CODE REVIEW

**Comment by lbanner** [comment]
@brandonPurvis Was this based on a GH Issue? If so -- can you note the GH Issue # in your description? Thanks!

**Comment by brandonPurvis** [comment]
I found this bug, should I also make an issue? @lbanner

**Comment by lbanner** [comment]
No worries -- just wanted to make sure it was tied to an issue if already existed -- You're good with this one. Thanks!

File: website/util/sanitize.py [hunk 1]
```diff
@@ -99,6 +99,19 @@ def safe_unescape_html(value):
     return value
 
 
+# TODO: Remove safe_unescape_html when mako html safe comes in
+def safe_unescape_html_in_results(func):
```
Comment (abought) [inline website/util/sanitize.py]: What is the advantage of this decorator? It seems like it reduces clarity, and doesn't save a lot of boilerplate.

(In fact, the name you have to type is longer than if you'd called `safe_unescape_html` directly)

**Comment by abought** [comment]
Before turning off escaping, I like to understand the origin of the issue. (so as to avoid side effects)

I notice that the search page is built on knockout. Is it fair to assume that knockout is making characters HTML-safe, but they are already being stored in the database as &amp; escape sequences?

By running the OSF on your own computer, this should be easy to test- just fill in the schools field and use a mongo shell (or robomongo) to look at how the user record is being stored in the DB.

**Comment by abought** [comment]
And... confirmed: the artifacts on search and profile pages are because it's stored in the database as &amp; .

**Comment by sloria** [comment]
Closing for now until this gets back on the priority queue.

**Pull request closed without merging**